mk-poetry-dep.nix

```diff
@@ -152,14 +152,18 @@ pythonPackages.callPackage
       # We need to retrieve kind from the interpreter and the filename of the package
       # Interpreters should declare what wheel types they're compatible with (python type + ABI)
       # Here we can then choose a file based on that info.
-      src =
+      src = let
+          # tagRef = if (sourceSpec ? tag) then "refs/tags/${sourceSpec.tag}" else null;
+        in
         if isGit then
           (
-            builtins.fetchGit {
+            builtins.fetchGit (lib.debug.traceValSeq ({
```

Comment:
This should be not left it no, no?

Reply:
yes it should go away :)  I am not sure this whole PR is a definitive fix nor how to operate/update the tests so I dont intend to complete this as of now. Anyone feel free to pick it up.